go/tricorder/doc.go

```diff
@@ -184,9 +184,19 @@ If code generates a metric's value, register the callback function like so
 		tricorder.None,
 		"generated int description")
 
-Tricorder can collect a distribution of values in a metric. In distributions,
-values are always floats. With distributions, the client program must manually
-add values. Distribution instances are safe to use from multiple goroutines.
+Tricorder can collect a distribution of values in a metric.
+With distributions, the client program must manually add values.
+Although Distributions store values internally as floats, they can
```

Comment:
I think it should be made clear that internally metrics are stored as float64 (rather than just saying float).